net/raft/dev/run-bootstrapped-node.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Enables job control to allow backgrounding & foregrounding cored process
+set -m
```

Comment:
I think we also want `set -e` in these scripts. And maybe also `set -o pipefail` for good measure.